label_sync/cluster/label_sync_job.yaml

```diff
@@ -26,10 +26,10 @@ spec:
       restartPolicy: Never  # https://github.com/kubernetes/kubernetes/issues/54870
       containers:
       - name: label-sync
-        image: gcr.io/k8s-testimages/label_sync:v20171215-477dc7ebc
+        image: gcr.io/k8s-testimages/label_sync:v20180227-377685119
         args:
         - --config=/etc/config/labels.yaml
-        - --dry-run=false
+        - --confirm=false
```

Comment:
=true?

Reply:
whoops thought I had pushed that, guess it was still local